client/my-sites/stats/controller.jsx

```diff
@@ -102,11 +104,36 @@ function getSiteFilters( siteId ) {
 			id: 'stats-year',
 			period: 'year',
 		},
+		{
+			title: i18n.translate( 'WordAds - Days' ),
+			path: '/stats/wordads/day/' + siteId,
+			period: 'day',
+		},
+		{
+			title: i18n.translate( 'WordAds - Weeks' ),
+			id: 'stats-wordads-week',
+			period: 'week',
+		},
+		{
+			title: i18n.translate( 'WordAds - Months' ),
+			id: 'stats-wordads-month',
+			period: 'month',
+		},
+		{
+			title: i18n.translate( 'WordAds - Years' ),
+			id: 'stats-wordads-year',
+			period: 'year',
+		},
```

Comment:
Idea for a followup PR: it seems that all the filter arrays in Stats are not needed at all. The only properties that are actually used are `period` and `altPaths` and the others (`title`, `id`, ...) are not. Maybe it made sense in some past version and someone forgot to remove the dead code?